python/demo/helmholtz2D/demo_helmholtz_2d.py

```diff
@@ -15,8 +15,8 @@
 import numpy as np
 from mpi4py import MPI
 
-from dolfinx import (Function, FunctionSpace, UnitSquareMesh,
-                     has_petsc_complex, solve)
+from dolfinx import Function, FunctionSpace, UnitSquareMesh, has_petsc_complex
+from dolfinx.fem import LinearProblem
```

Comment:
I would like to see `fem.LinearProblem` used the the demos. Using namespacing means we can keep names short, like 'LinearProblem', without losing the context.